packages/app/src/device-manager/services/device-manager.service.ts

```diff
@@ -176,6 +180,7 @@ export class DeviceManager {
     this.currentDeviceInitializationFailed = true
   }
   private deviceConnectedListener = () => {
+    //this do a lot of important stuff :D
```

Comment:
Unnecessary comment 😉 

If you see an area for improvement, please add a TODO comment above the method. It's nice if a TODO has a link to the task & description.